src/librbd/api/Pool.cc

```diff
@@ -246,6 +246,11 @@ int Pool<I>::init(librados::IoCtx& io_ctx, bool force) {
     return r;
   }
 
+  r = io_ctx.create(RBD_TRASH, false);
+  if (r < 0) {
+    return r;
```

Comment:
It would be nice to emit "failed to create trash" error here and "failed to enable RBD application" error above.